packages/components/pie-card-container/src/index.ts

```diff
@@ -21,8 +21,8 @@ const componentSelector = 'pie-card-container';
  */
 export class PieCardContainer extends LitElement implements CardContainerProps {
     @property()
-    @validPropertyValues(componentSelector, interactionTypes, 'none')
-    public interactionType: CardContainerProps['interactionType'] = 'none';
+    @validPropertyValues(componentSelector, interactionTypes, 'button')
```

Comment:
since none was removed now, do you think we should rename `interactionTypes` to `tags` to align with the link component and enforce consistenty?